### ISSUE TITLE
pypi lists textual as compatible with 3.7 but it needs at least 3.8

### ISSUE THREAD

**Issue by danmur** [willmcgugan/textual#31]
I can't get the traceback because it destroys my terminal but in layout.py here:

```
    @classmethod
    def _assemble_chops(
        cls, chops: list[dict[int, list[Segment] | None]]
    ) -> Iterable[list[Segment]]:

        for bucket in chops:
            yield sum(
                (segments for _, segments in sorted(bucket.items()) if segments),
                start=[],
            )
```

sum() can't take start as a keyword argument before 3.8

**Comment by danmur** [comment]
docs: https://docs.python.org/3/library/functions.html#sum

**Comment by willmcgugan** [comment]
I'll fix that in the next version

**Comment by willmcgugan** [comment]
Fixed in 0.1.7
